### PR TITLE
COMMERCE-1056

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test

File: commerce-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -224,11 +224,13 @@ gtin-help=Enter the Global Trade Item Number (GTIN). These identifiers include U
 gtin-x=GTIN: {0}
 image-extensions=Allowed Image Extensions
 image-max-size=Image Maximum Size
+import-regions=Import Regions
 inventory=Inventory
 inventory-engine=Inventory Engine
 inventory-engine-method=Inventory Engine Method
 invite-user=Invite user
 invite-user-to-this-account=Invite User to This Account
+italy=Italy
```
Comment (ethanbustad) [inline commerce-lang/src/main/resources/content/Language.properties:233]: we should use the OOTB portal keys for this -- `country.italy`, `country.united-states`, etc.

**Review by ethanbustad — CHANGES_REQUESTED** [review]
Just the language thing, and the rebase. I tried to rebase it locally, but there've been significant changes to this package since the PR was made, so the conflicts are pretty nasty. Takes a bit of reworking. Just a heads-up.

**Comment by ethanbustad** [comment]
Rebased and sent along to https://github.com/liferay/com-liferay-commerce/pull/1787. Note that I also adjusted the starters a bit to match changes that've been made in that package since this PR was sent, and switched to using portal language keys.

**Pull request closed without merging**